src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.netcoreapp.cs

```diff
@@ -41,6 +41,19 @@ public static void CreateServer_ConnectClient()
             }
         }
 
+        [Fact]
+        public static void CreateServer_ConnectClient_UsingUnixAbsolutePath()
+        {
+            string name = Path.Combine("/tmp", GetUniquePipeName());
+            using (var server = new NamedPipeServerStream(name, PipeDirection.InOut, 1, PipeTransmissionMode.Byte, PipeOptions.CurrentUserOnly))
+            {
+                using (var client = new NamedPipeClientStream(".", name, PipeDirection.InOut, PipeOptions.CurrentUserOnly))
+                {
+                    client.Connect();
+                }
+            }
+        }
```

Comment:
Since folks will probably do it, and we won't want to regress them, I think it'd be good also to have a Unix-only test that validates we can connect on Unix between a NamedPipeServer/ClientStream and a Socket created with a UnixDomainSocketEndPoint.

Reply:
Since you mention this case: my idea was to not add support between NamedPipe and abstract path UnixDomainSocketEndPoint, but that is also possible. If we want that I can perhaps do it now, or open an issue to track it.

Reply:
> my idea was to not add support between NamedPipe and abstract path UnixDomainSocketEndPoint

My point was that I think you've already done it (not "abstract" though in the domain socket sense), and as such we would ideally have a test that we don't regress it.